dist/server/BackwardCompat.js

```diff
@@ -0,0 +1,167 @@
+/**
+ * Silex, live web creation
+ * http://projects.silexlabs.org/?/silex/
+ *
+ * Copyright (c) 2012 Silex Labs
+ * http://www.silexlabs.org/
+ *
+ * Silex is available under the GPL license
+ * http://www.silexlabs.org/silex/silex-licensing/
+ */
+
+const Path = require('path');
+const constants = require('./Constants.json');
+
+/**
+ * the version of the website is stored in the generator tag as "Silex v-X-Y-Z"
+ * used for backward compat
+ * also the static files are taken from //{{host}}/static/Y-Z
+ */
+const LATEST_VERSION = require(Path.resolve(__dirname, '../../package.json')).version.split('.');
```

Comment:
FYI, if you start the server with NPM, it sets an env variable for you : `process.env.npm_package_version`